tiledb/sm/group/group.cc

```diff
@@ -57,7 +60,10 @@ class GroupStatusException : public StatusException {
   }
 };
 
-Group::Group(const URI& group_uri, StorageManager* storage_manager)
+Group::Group(
+    ContextResources& resources,
+    const URI& group_uri,
+    StorageManager* storage_manager)
     : group_uri_(group_uri)
     , storage_manager_(storage_manager)
     , config_(storage_manager_->config())
```

Comment:
Note for the future: We'll need to initialize this variable from something other than the storage manager at some point.